piksi_multi_cpp/src/sbp_callback_handler/sbp_callback_handler_relay/sbp_callback_handler_relay_imu_raw.cc

```diff
@@ -0,0 +1,31 @@
+#include "piksi_multi_cpp/sbp_callback_handler/sbp_callback_handler_relay/sbp_callback_handler_relay_imu_raw.h"
+
+namespace piksi_multi_cpp {
+// Define ROS and SBP message types.
+typedef piksi_rtk_msgs::ImuRawMulti ROSMSgType;
+typedef msg_imu_raw_t SBPMSgType;
```

Comment:
potentially dangerous as typdefs are in namespace and not class - would try to avoid